component_documentation/componentDoc.html

```diff
@@ -853,7 +853,113 @@ <h2 class="card__heading">Boat & the River</h2>
 
     </div>
      <!-- card ends here -->
+    
+      <!-- input tabs start from here -->
+    <div class="component-container" id="input--component">
+        <h1 class="component__heading">Input Fields</h1>
+        <p class="component__sub-heading">Sample is used to represent the profile pictures and icons. Monad UI has different sizes of Avatar.</p>
+        <div class="sub-component-container">
+            <h2 class="sub-component__heading">Responsive Input fields</h2>
+           <div class="component-display">
+            <input type="text" class="input" placeholder="primary">
+            <input type="text" class="input input--success" placeholder="success" >
+            <input type="text" class="input input--danger" placeholder="danger">
+            <input type="text" class="input input--warning" placeholder="warning">
+           </div>
+           <div class="component-code">
+               <div class="code-snippet-container ">
+                   <textarea class="code-snippet" name="code-snippet" rows="5" cols="" readonly>
+<input type="text" class="input" placeholder="primary">
+<input type="text" class="input input--success" placeholder="success" >
+<input type="text" class="input input--danger" placeholder="danger">
+<input type="text" class="input input--warning" placeholder="warning">
+                   </textarea>
```

Comment:
try proper structure here

Reply:
the structure is not proper here because it is for the code snippet display. If it will be arranged properly here, then on the UI it will not look good. That's why I have to do that.